packages/tools/kadena-cli/src/account/utils/accountHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import yaml from 'js-yaml';
+import { readdirSync } from 'node:fs';
+import { join } from 'node:path';
+
+import { ACCOUNT_DIR } from '../../constants/config.js';
+import { services } from '../../services/index.js';
+import type { IAccountAliasFile } from '../types.js';
+import { isEmpty } from '../utils/addHelpers.js';
+
+interface IAccount {
+  name: string;
+  alias: string;
+}
+
+const readAccountFromFile = async (accountFile: string): Promise<IAccount> => {
+  const accountAlias = accountFile.split('.')[0];
+  const content = await services.filesystem.readFile(
+    join(ACCOUNT_DIR, accountFile),
+  );
+  const account =
+    content !== null ? (yaml.load(content) as IAccountAliasFile) : null;
+  return {
+    name: account?.name ?? '',
+    alias: accountAlias,
+  };
+};
+
+export async function ensureAccountExists(): Promise<void> {
+  if (!(await services.filesystem.directoryExists(ACCOUNT_DIR))) {
+    console.error(`No account created yet. Please create an account first.`);
+    process.exit(1);
```

Comment:
Don't `process.exit()` in helpers, or preferably at all. I know it's been used in some other places but they need to go away as well.  If there is an error just throw.

Reply:
Also no console output in helpers.